filebeat/docs/multiline.asciidoc

```diff
@@ -46,10 +46,9 @@ To consolidate these lines into a single event in Filebeat, use the following mu
 
 [source,json]
 -------------------------------------------------------------------------------------
-multiline:
-  pattern: '^[[:space:]]'
-  negate: false
-  match: after
+multiline.pattern: '^[[:space:]]'
```

Comment:
I'm also wondering if it would help to show more of the config file so users can see where  sub-fields are defined (we can leave out sections and use ..., but some context might help). I'm not suggesting you should fix this now, but if you think it's a good idea, let me know, and I'll create an issue to track the work.